lib/grp.gd

```diff
@@ -4495,12 +4495,12 @@ DeclareGlobalFunction("GroupEnumeratorByClosure");
 ##
 ##  <#GAPDoc Label="LowIndexSubgroups">
 ##  <ManSection>
-##  <Oper Name="LowIndexSubgroups"
-##   Arg='G, index'/>
 ##  <Oper Name="LowIndexSubgroups" Arg='G, index'/>
+##  <Oper Name="LowLayerSubgroups" Arg='G, layer'/>
 ##
 ##  <Description>
-##  These operations computes representatives of the conjugacy classes of
+##  The operation <C>LowIndexSubgroups</C> computes representatives of the
```

Comment:
Actually GAPDoc recommends to use `<Ref>` inside `<ManSection>` even if it refers to the object documented in this ManSection: see example here:

https://github.com/frankluebeck/GAPDoc/blob/6108fcbca047c8b6484c512c3ba5c4bdd5bb8b7c/example/example.xml#L109-L118

GAPDoc will not expand the reference in this case. IIRC this is useful because if you copy and paste the text elsewhere, `<Ref>` will be there already, and also in case the manual layout will display `<Ref>` and `<C>` elements differently. Perhaps there are actually deeper reasons, which @frankluebeck may tell.